### ISSUE TITLE
Create new project button on 404 leads to the wrong place.

### ISSUE THREAD

**Issue by flukeout** [mozilla/thimble.mozilla.org#1716]
Currently it links to ``https://thimble.mozilla.org/editor`` but should link to ``https://thimble.mozilla.org/projects/new``.

**Comment by raghavaggsss** [comment]
I am interested in working on this bug! Please let me know how to proceed.

**Comment by humphd** [comment]
@cadecairos is this something that you have to fix on S3 or is it in our code?  I can't remember.  If it's fixable by @raghavaggsss, maybe someone could point the way?  If not, let's remove the `good-first-bug` label.

**Comment by cadecairos** [comment]
This is a static file uploaded to S3. Whoever works on it can upload the completed HTML file in this issue and I can upload it to the bucket.

**Comment by humphd** [comment]
Where is the source of the current one, so that this modification could happen?

**Comment by cadecairos** [comment]
It's not under version control (lol): `view-source:http://s3.amazonaws.com/publish-webmaker-org-staging-s3bucket-bqo25c3iqx2o/error.html`

Future project: put it in the publish repo and have it automatically update S3 on a deploy?

**Comment by humphd** [comment]
Let's morph this bug into that.  @gideonthomas is probably the right person to drive it.

**Comment by flukeout** [comment]
Sorry @raghavaggsss - I've removed the Good First Bug label. Since this file isn't under source control and requires @cadecairos to manually update it, I'll just get it done quickly myself. Thank you, though!

**Comment by raghavaggsss** [comment]
@flukeout No problem!

**Comment by gideonthomas** [comment]
TODO:
- Add error page to publish.wm.org and update S3 error.html page via Travis.
- Update "Create new project button" on error page to redirect to `/projects/new`

**Comment by Pomax** [comment]
@gideonthomas should these be part of this issue, or issues in their own right?

**Comment by flukeout** [comment]
I think we should break this out into new issues, if there are any. It's a bit confusing.

**Comment by gideonthomas** [comment]
I don't think we need to! The second one is a one line fix that I will be doing in the same patch. So, the only issue is the first TODO. I just broke it down into two so that I'll remember to add that change to my patch
